### DISCUSSION TITLE
No IP:Port displayed in Unraid with MacVlan

### DISCUSSION THREAD

**Discussion by Demthios** [louislam/dockge#602] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Please excuse me as I'm new to docker/compose. When I set up a docker and attach it to my br1 network which is a dedicated network port on my unraid box, setup as MacVlan, and get the docker running it displays a super long random character network id instead of br1. It also doesn't show any IP:ports in the Unraid console. But if I attach it to a custom dockernetwork I've created in bridge mode it does show the correct IP:port in Unraid. How can I Fix this? 

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

Unraid

### 🌐 Browser

Chrome

### 🐋 Docker Version

Manyfold 

### 🟩 NodeJS Version

0.76.1

**Comment by trythatagain** [comment]
So you've created a macvlan network, is that within the 'compose.yml' file for the container or via the CLI?
I don't use Unraid, so not sure what shows on the dashboard, but what do you see when looking at the container in Dockge? I see 'Running' and a list of ports.

It might be worth posting your compose.yml file.
